src/main/java/com/twilio/TwilioBearerTokenAuth.java

```diff
@@ -17,20 +18,22 @@ public class TwilioBearerTokenAuth {
     private static String region = System.getenv("TWILIO_REGION");
     private static String edge = System.getenv("TWILIO_EDGE");
     private static volatile BearerTokenTwilioRestClient restClient;
+    private static TokenManager tokenManager;
 
     private static volatile ExecutorService executorService;
 
     private TwilioBearerTokenAuth() {
     }
 
-    public static synchronized void init(final String accessToken) {
```

Comment:
are we expecting the client to first create a token manager and then pass it here. We should just collect all the info from the client need to initialize and internally create/set the tokenManger if needed